bindings/wasm/examples/README.md

```diff
@@ -1,67 +1,45 @@
 
-- You can then run each example with:
-    ```bash
-    npm run example:node -- <example_name>
-    ```
+```bash
+npm run example:stardust -- <example-name>
+```
 
-- For instance, to run the `create_did` example use:
-    ```bash
-    npm run example:node -- create_did
-    ```
+For instance, to run the `ex0_create_did` example execute:
 
-The following examples are currently available:
+```bash
+npm run example:stardust -- ex0_create_did
+```
 
-|  #  | Name                                      | Information                                                                                                                                                                                                                                |
-| :-: | :---------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|  1  | [create_did](src/create_did.js)           | Generates and publishes a DID Document, the fundamental building block for decentralized identity.                                                                                                                                         |
-|  2  | [manipulate_did](src/manipulate_did.js)   | Add verification methods and service endpoints to a DID Document and update an already existing DID Document.                                                                                                                              | |
-|  3  | [resolve_history](src/resolve_history.js) | Advanced example that performs multiple updates and demonstrates how to resolve the DID Document history to view them.                                                                            |
-|  4  | [resolve_did](src/resolve_did.js)          | Resolves an existing DID to return the latest DID Document.                                                                                                                                                                                |
-|  5   | [key_exchange](src/key_exchange.js) | Demonstrates Elliptic-curve Diffie-Hellman (ECDH) cryptographic key exchange with DID Documents. | |
-|  6   | [private_tangle](src/private_tangle.js)   | Showcases the same procedure as `create_did`, but on a private tangle - a locally running hornet node.                                                                                                                                     |
+| #   | Name                                            | Details                                                          |
+|-----|-------------------------------------------------|------------------------------------------------------------------|
+| 0   | [ex0_create_did](src/ex0_create_did.ts)         | Create a DID Document and publish it in a new Alias Output.      |
```

Comment:
The Wasm examples should be renamed `<digit>_<name>` in accordance with the Rust ones.

Reply:
Updated #984 to include this task, can be addressed in a separate PR.